charts/scalardl/values.yaml

```
@@ -227,3 +227,9 @@ ledger:
 
   # -- Name of existing secret to use for storing database username and password
   existingSecret: ""
+
+  serviceAccount:
+    # -- Name of the existing service account resource
+    serviceAccountName: ""
+    # -- Specify to mount secret or not
```

Should we say `service account` for consistency?
```suggestion
    # -- Specify to mount a service account token or not
```

---

Thank you for your suggestion!
I think it is better than the current description.
So, I will apply it. Thank you!